bin/fpm.sh

```diff
@@ -93,6 +93,7 @@ for module in "$modules"; do
     modules/api/target/classes/api.properties=${home}/conf/api.properties\
     modules/api/target/generated/swagger-ui/=${home}/content/ui/dist/swagger/swaggerjson\
     modules/assignment/target/classes/assignment.properties=${home}/conf/assignment.properties\
+    modules/assignment/target/classes/ehcache.xml=${home}/conf/ehcache.xml\
```

Comment:
Do we need a corresponding update on wasabi-intuit [cc: @jcwuzoegiver] to include ehcahce.xml there?